driver.cc

```diff
@@ -207,13 +206,10 @@ open_pipe(void)
 			st_glob.outp = stdout;
 	}
 }
-/******************************************************************************/
-/* PRINT PROMPT FOR AN INPUT MODE                                             */
-/******************************************************************************/
+
+// Prints a prompt for an input mode.
```

Comment:
```suggestion
// Print a prompt for an input mode.
```
For consistency with other comments

Reply:
Done.

In the future, I'd like to migrate towards using the commenting convention from the Google C++ style guide (which I'm most familiar with from when I worked at Google) that prefers using verb phrases.  But that's going to be a large overhaul.